tdrs-frontend/src/components/Footer/Footer.test.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react'
-import { mount } from 'enzyme'
+import { render } from '@testing-library/react'
```

Comment:
I am converting all our tests to use react-testing-library when possible for improved developer experience.